config/database.yml

```diff
@@ -79,8 +79,18 @@ test:
 # Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
 # for a full overview on how database connection configuration can be specified.
 #
+# production:
+#   <<: *default
+#   database: board_game_geek_data_production
+#   username: board_game_geek_data
+#   password: <%= ENV["BOARD_GAME_GEEK_DATA_DATABASE_PASSWORD"] %>
+
 production:
   <<: *default
-  database: board_game_geek_data_production
-  username: board_game_geek_data
-  password: <%= ENV["BOARD_GAME_GEEK_DATA_DATABASE_PASSWORD"] %>
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  database: board_game_data_aws
+  username: postgres
+  password: bgb2307!
+  host: board-game-data-aws.ch6oocyqeqsl.us-east-2.rds.amazonaws.com
```

Comment:
hey do we need to include our db URL also ... theres an exposed password that we might want to encrypt! 

Wanna hop on a zoom all together and check this out?

Reply:
Let's do it! Teamwork will make the dream work.

Reply:
Good call! I'll add it to the rails credentials